### ISSUE TITLE
[Bug]: If multiples actions or traits have the same description, only the last one is displayed

### ISSUE THREAD

**Issue by freohr** [javalent/fantasy-statblocks#228]
### Check for existing bug reports before submitting.

- [X] I searched for existing [Bug Reports](https://github.com/javalent/fantasy-statblocks/issues?q=is%3Aopen+label%3Abug+sort%3Aupdated-desc) and found no similar reports.

### Expected Behavior

When writing multiples actions or traits, they should all be displayed as long as they have different names, whichever description they might have.

### Current behaviour

If multiple actions or traits have different names but the same exact description, only the last one with this description gets displayed.

### Reproduction

Here's a minimalist statblock that can reproduce this issue

````
```statblock
layout: Basic 13th Age Monster Layout
actions:
    - name: Action 1
      desc: 2 damage
    - name: Action 2
      desc: 3 damage
    - name: Action 3
      desc: 3 damage
traits:
    - name: Trait 1
      desc: description 1
    - name: Trait 2
      desc: description 2
    - name: Trait 3
      desc: description 2
```
````

and the displayed result

![image](https://github.com/javalent/fantasy-statblocks/assets/3462951/3a7a1fd5-a987-4cfd-bf38-74fb294efaa2)

### Which Operating Systems are you using?

- [ ] Android
- [ ] iPhone/iPad
- [X] Linux
- [ ] macOS
- [ ] Windows

### Obsidian Version Check

	Obsidian version: v1.3.3 	Installer version: v1.2.8

### Plugin Version

3.6.5

### Confirmation

- [X] I have disabled all other plugins and the issue still persists.

**Comment by sigrunixia** [comment]
Hey @freohr This should be fixed in #249. :) Can you confirm just in case?

**Comment by freohr** [comment]
> Hey @freohr This should be fixed in #249. :) Can you confirm just in case?

@sigrunixia Yes, it works correctly now, thank you :)

![image](https://github.com/javalent/fantasy-statblocks/assets/3462951/96322c57-8f06-4c03-8257-23ab597a6389)
